### ISSUE TITLE
license.type does nothing

### ISSUE THREAD

**Issue by creynders** [assemble/verb#37]
Sorry for flooding you with issues, but verb's really promising and I'd _really_ like to use it, but it appears there's quite a lot of bugs.

If I try to use `license.type` as shown on verbiage.io

```sh
## License
Copyright (c) {%= date('YYYY') %} {%= author.name %}, contributors.
Released under the {%= license.type %} license
```

It doesn't appear to do anything. However I think it's due to `license` being overridden by verb, since if I use `{%= license %}` it outputs:

```js
function bound() {
        // `Function#bind` spec
        // http://es5.github.io/#x15.3.4.5
        if (partialArgs) {
          // avoid `arguments` object deoptimizations by using `slice` instead
          // of `Array.prototype.slice.call` and not assigning `arguments` to a
          // variable as a ternary expression
          var args = slice(partialArgs);
          push.apply(args, arguments);
        }
        // mimic the constructor's `return` behavior
        // http://es5.github.io/#x13.2.2
        if (this instanceof bound) {
          // ensure `new bound` is an instance of `func`
          var thisBinding = baseCreate(func.prototype),
              result = func.apply(thisBinding, args || arguments);
          return isObject(result) ? result : thisBinding;
        }
        return func.apply(thisArg, args || arguments);
      }
```

I assume it's the binding code for `license()`? Probably it would be a good idea to set up the binding first and then merge it with the data from package.json's `license`.

**Comment by jonschlinkert** [comment]
sorry! I think the problem is just verbiage.io. I didn't know that was live! It's a documentation issue.

Here is the helper that is responsible https://github.com/helpers/helper-license.

You want to use it as a function:

```js
{%= license() %}
```

If you look at the `.verb.md` file in that project you can see how it's used in the wild too.

> Sorry for flooding you with issues,

absolutely no problem! I don't mind issues at all. The biggest thing we struggle with is documentation, since it's usually so disconnected from the code itself, it's hard to remember to update - ironically, I created verb to solve this lol, and I planned on publishing the docs to verbiage.io but I honestly don't remember actually doing it lol. 

Let me know if this helps

**Comment by jonschlinkert** [comment]
btw, if license doesn't do what you need, let me know more about your use case, I'd be happy to get it working for you

**Comment by creynders** [comment]
Hi @jonschlinkert `license()` is ok, but in some case it'd be useful to be able to access `license.type` and `license.url` separately. Also, it would be more consistent, since all the other values in `package.json` are directly accessible as well.

**Comment by jonschlinkert** [comment]
To use any value from package.json, just do `{%= foo %}`, and it should work now. It always should have, but the license helper in particular was conflicting with the license property on package.json.

This (approach) is fixed in 0.7.0.
